### PR TITLE
Update tcmu-runner's log directory using env. variable

### PR DESCRIPTION
**Pull request by SaravanaStorageNetwork**
Update tcmu-runner's log directory using env. variable

Signed-off-by: Saravanakumar <sarumuga@redhat.com>

---
This change is [Reviewable](https://reviewable.io/reviews/gluster/gluster-kubernetes/337)

### CODE REVIEW

**Comment by jarrpa** [comment]
Needs a kube-templates counterpart. :)

File: deploy/kube-templates/glusterfs-daemonset.yaml [hunk 1]
```diff
@@ -104,3 +107,8 @@ spec:
       - name: glusterfs-ssl
         hostPath:
           path: "/etc/ssl"
+parameters:
+- name: TCMU_LOG_DIR
+  displayName: Tcmu runner log directory
+  description: This value is to set tcmu runner log directory
+  value: /var/log/glusterfs/gluster-block
```
Comment (jarrpa) [inline deploy/kube-templates/glusterfs-daemonset.yaml]: Kubernetes has no notion of Templates, that's an OpenShift thing, thus the `parameters` field is invalid. Please remove.

**Comment by SaravanaStorageNetwork** [comment]
> Needs a kube-templates counterpart. :)

Please clarify about this. Where is the kube-templates present? 

ps: I mistook this for changes in glusterfs-daemonset.yaml.

**Comment by jarrpa** [comment]
Oh, no, the first part was right! The env variable was correct, it's just that you can't specify a parameters section.

**Comment by jarrpa** [comment]
This will require some changes to gk-deploy for proper integration, too. We should add a command line option to configure thus and a global car to default and overwrite with the cli option.

**Comment by jarrpa** [comment]
Thinking on it a bit further, just hardcode the new env var to `/var/log/glusterfs/gluster-block` and we can add the gk-deploy configuration later.

**Comment by SaravanaStorageNetwork** [comment]
Added in both ocp and kube template.

**Comment by jarrpa** [comment]
You misunderstood my earlier comment, I only wanted the `kube-templates` file changed. Please ensure the following:

* `deploy/kube-templates/glusterfs-daemonset.yaml` should list the variable and a hard-coded value of `15`.
* `deploy/ocp-templates/glusterfs-daemonset.yaml` should list the variable with a `"${TCMU_LOG_DIR}"` (note the quotes) parameter, and there should also be a parameter entry at the bottom with the default value of `"/var/log/glusterfs/gluster-block"`

Also, please squash all your changes into one commit then do a `git push -f` to overwrite the branch. We don't need to see everything fixed for the patch, just the final patch. :)

**Comment by jarrpa** [comment]
@SaravanaStorageNetwork LGTM! Rebase and resolve merge conflicts from #335.

**Comment by jarrpa** [comment]
@SaravanaStorageNetwork Don't know when you did the update, but next time please leave a comment. I don't get alerts for commit pushes. :)

**Comment by SaravanaStorageNetwork** [comment]
@jarrpa  Sure - will leave comment in future 👍

**Pull request merged**